checks.d/win32_event_log.py

```diff
@@ -190,6 +192,15 @@ def _wmi_to_ts(self, wmi_ts):
                       second=second, microsecond=microsecond) + tz_delta
         return int(calendar.timegm(dt.timetuple()))
 
+    def _tags(self, event, tags):
```

Comment:
Why not just pass the `EventCode` as argument ? (since you don't use/need `event`)